tests/integration/test_valid_data_in_request.py

```diff
@@ -20,22 +20,10 @@ def test_valid_data_in_request(test_env, default_request):
     assert response.status == 200
     assert response.data["state"] == "complete"
 
-    response_dependencies = list_of_dict_to_list_of_name_type_version(response.data["dependencies"])
+    response_dependencies = make_list_of_packages_hashable(response.data["dependencies"])
     expected_dependencies = test_env["get"]["dependencies"]
-    assert response_dependencies == expected_dependencies
+    assert sorted(response_dependencies) == sorted(expected_dependencies)
```

Comment:
Sorting of `response_dependencies` was already done in the function.

Reply:
Done, Thank you!